docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -337,71 +373,212 @@ apiVersion: "observability.openshift.io/v1"
+      type: openShiftLabels
+      openShiftLabels:
+        foo: bar
   pipelines:
-  - name: application-logs
-    filterRefs:
-    - detectexception
-    - labels
-    - parse-json
+    - name: app-logs
+      filterRefs:
+      - my-multiline
+      - my-parse
+      - my-labels
+----
+NOTE: Drop filter, Prune filter and KubeAPIAudit filters remain unchanged
+[source, yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  filters:
+    - name: drop-debug-logs
+      type: drop
+      drop:
+      - test:
+        - field: .level
+          matches: debug
+    - name: prune-fields
+      type: prune
+      prune:
+        in:
+        - .kubernetes.labels.foobar
+        notIn:
+        - .message
+    - name: audit-logs
+      type: kubeAPIAudit
+      kubeAPIAudit:
+        omitResponseCodes:
+        - 404
+        - 409
 ----
 ==== Validation & Status
 Most validations are enforced when a resource is created or updated which provides immediate feedback.  This is
 a departure from previous releases where all validation occurred post creation requiring inspection of the resource status location.  Some validation still occurs post resource creation for cases where is not possible to do so at creation or update time.
 
-Instances of the **ClusterLogForwarder.observability.openshift.io** must satisfy the following conditions before the operator will deploy the log collector: Authorized, Valid, Ready.  An example of these conditions is:
+Instances of the **ClusterLogForwarder.observability.openshift.io** must satisfy the following before the operator will deploy the log collector:
+
+*Conditions:* `Authorized, Valid, Ready` = "True"
+
+*Spec Validations:* `Filters, Inputs, Outputs, Pipelines` = True
 
-.6.0 Status Conditions
+.6.0 Status "True" Conditions Example
 [source, yaml]
 ----
 apiVersion: "observability.openshift.io/v1"
 kind: ClusterLogForwarder
 status:
   conditions:
-  - message: ""
-    status: "True"
-    type: Ready
-  - message: "permitted to collect log types: [application]"
-    reason: ClusterRoleExists
-    status: "True"
-    type: observability.openshift.io/Authorized
-  - message: ""
-    reason: Validation Success
-    status: "True"
-    type: observability.openshift.io/Valid
-  inputs:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-application
-  outputs:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-rh-loki
-  pipelines:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-application-logs
+    - message: "permitted to collect log types: [application]"
+      reason: ClusterRoleExists
+      status: "True"
+      type: observability.openshift.io/Authorized
+    - message: ""
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid
+    - message: ""
+      status: "True"
+      type: observability.openshift.io/Ready
+  filtersStatus:
```

Comment:
These become xxxConditions

Reply:
Do you think we need additional details or a note otherwise, in this example?

Reply:
I think we just need the correct fieldName which will be xxxxConditions once https://github.com/openshift/cluster-logging-operator/pull/2762 merges